src/cnn.py

```diff
@@ -139,21 +119,19 @@ def get_keras_model(image_shape, label_encoder):
 
     print(model.summary())
 
-    nadam = keras.optimizers.Nadam()
+    nadam = optimizers.Nadam()
 
+    # TODO Unused, reimplement this?
     cdl_indices_to_mask = label_encoder.transform(CDL_CLASSES_TO_MASK)
```

Comment:
Explain the idea behind masking in the readme:  CDL developed contains roads, buildings, and (at 30 meter resolution) many NAIP pixels that are neither roads nor buildings, hence the idea of masking out CDL developed when computing loss.